app/views/common/_event_social_buttons.html.erb

```diff
@@ -19,8 +19,7 @@
 	</div>
 
 	<div>
-		<a class='button--circle--large grey'>
-			<!--= on 'click' (open_modal_if_confirmed 'shareEmail') -->
+		<a class='button--circle--large grey' href='mailto:?subject=Check the <%= @event.name %> event from <%= @nonprofit.name %>!&amp;body=Hi,%0D%0A%0D%0A<%= @event.summary.present? ? @event.summary.concat("%0D%0A%0D%0A") : ''%>Follow this link for the event and participate: <%= url %>'>
```

Comment:
Oh my goodness, no, that's a recipe for disaster. Use [mail_to](https://api.rubyonrails.org/v4.2/classes/ActionView/Helpers/UrlHelper.html#method-i-mail_to)